docs-website/docusaurus.config.js

```diff
@@ -69,6 +71,26 @@ module.exports = {
           label: "Roadmap",
           position: "right",
         },
+        {
+          type: "dropdown",
+          label: "v0.10.4",
```

Comment:
this should always be "latest"

Reply:
Or ideally - when building from vercel, it has the version number, and when building as part of our docs generation, it says “latest”